docs/connecting-data/connecting-to-apis-and-external-data/datadog-api.md

```diff
@@ -2,6 +2,151 @@
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the Datadog API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+![Datadog Form][image-1]
+
+### Name
+
+Enter any name that will help you recognize this data source from within your query window. &#x20;
+
+!!! info "Acceptable characters include"
+
+    * lowercase alphanumeric characters
+    * underscores
+
+### API Key
+
+=== "1. Sign In"
+
+    You can access your API Key at your [Datadog API Key Page](https://app.datadoghq.com/organization-settings/application-keys){target=_blank}. Sign in.
+    ![Sign in][image-8]
+
+=== "2. Find your API Key"
+
+    click on the API Key which will open a modal as in step two.
+    ![Generate API Key][image-4]
+
+=== "3. Copy your API Key"
+
+    In the open API Key modal copy the API Key
+
+    ![Copy API Key][image-7]
+
+### Application Key
+
+=== "1. Sign In"
+
+    You can access your Application Key at your [Datadog Application Key Page](https://app.datadoghq.com/organization-settings/application-keys){target=_blank}. Sign in.
+    ![Sign in][image-8]
+
+=== "2. Find your Application Key"
+
+    You can access your Application Key at your [Datadog Application Key Page](https://app.datadoghq.com/organization-settings/application-keys){target=_blank}.
+    
+    ![Copy Application Key][image-2]
+
+
+## Endpoints
+
+The table below shows a list of endpoints available to connect to within the DataDistillr application. If you need to connect to any endpoints not listed in the table below, please use the [Custom API](custom-apis.md) Form.
+
+| Endpoint     | Required Params | Optional  Params             | Description                              |
+|--------------|-----------------|------------------------------|------------------------------------------|
+| `synthetics` | public_id       | from_ts<br>to_ts<br>probe_dc | Returns the list of all Synthetic tests. |
+| `metrics`    | from            | host<br>tag_filter           | Returns active Metrics list.             |
+| `events`     | start<br>end    |                              | This endpoint queries the event stream.  |
+
+
+### Nav Tree
+
+The endpoint above will display as follows in the nav tree once your API has successfully connected.
+
+![Datadog Endpoints][image-3]
+
+## Sample Queries
+
+The following queries are intended to help you get started, and make life simpler querying within your API.
+
+For the following examples, suppose that my Datadog data source was called `mydatadog` and I want to query an endpoint. The endpoint goes after the Datadog data source name:
+
+!!! example "FROM Clause"
+
+    ```sql
+    FROM `mydatadog`.`<ENDPOINT>`
+    ```
+
+### Get Synthetics Endpoint
+
+Get the list of all Synthetic tests. This endpoint requires the Datadog `synthetics_read` authorization scope.
+
+```sql
+SELECT * FROM mydatadog.synthetics
```

Comment:
In the example above we show there must be back ticks, but in all the examples at the bottom there are no back ticks - which is correct for this API?